### DISCUSSION TITLE
Multi-modal inputs for a given ground sampling distance

### DISCUSSION THREAD

**Discussion by lzachmann** [Clay-foundation/model#313] [Q&A] [answered]
I noticed in the regression example a list of wavelengths that appears to have included bands from Sentinel-1, alongside the Sentinel-2 bands: https://github.com/Clay-foundation/model/blob/main/finetune/regression/biomasters_model.py#L67-L82. Let's say we were to add that back in.... Is Clay able to generate a single set of _n_=768 embeddings for a combined set of Sentinel-1 + Sentinel-2 inputs (on the assumption we create a new metadata entry that includes bands from both instruments)? Or is it preferred to compute the embeddings for each sensor separately?

**Comment by srmsoumya** [comment, marked as answer]
We can pass the combined chip of Sentinel-2 & Sentinel-1 as input to the model. I did try that with the Biomasters example but the loss curves were not very smooth, so I skipped Sentinel 1.
